CSharpMinifier/Rewriters/CSharpRewriter.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Rename;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace CSharpMinifier.Rewriters
+{
+    class CSharpRewriter : CSharpSyntaxRewriter
+    {
+        private SemanticModel _semanticModel;
+        private AdhocWorkspace _workspace;
+        private MinifierOptions _options;
+        private IdentifierGenerator _identifierGenerator; 
+
+        public CSharpRewriter(MinifierOptions options = null, bool visitIntoStructuredTrivia = true) : base(visitIntoStructuredTrivia)
+        {
+            _options = options ?? new MinifierOptions();
+            _identifierGenerator = new IdentifierGenerator();
+            //Add cause MSBuild does not work without it
+            var _ = typeof(Microsoft.CodeAnalysis.CSharp.Formatting.CSharpFormattingOptions);
```

Comment:
this decision is possible to avoid the exception of the type "The language 'C#' is not supported". Reason in the latest version of MSBuild, which does not load the required library, since it believes that it is not used in this program

Reply:
I mean add a website link with description to comment in the code.